src/components/ui/BottomTabIconLink.tsx

```diff
@@ -17,29 +24,42 @@ export const BottomTabIconLink: React.FC<Props> = ({ name, isSelected }) => {
   const path = mainRouteConfig[name].path;
   const title = mainRouteConfig[name].title;
 
+  const size: IconSize = 6;
+
+  const color: IconColor = isSelected ? 'primary' : 'gray-dark';
+
+  const className = 'mb-1';
   // Icon component based on "name" and "isSelected" variable
-  let iconName: IconName;
+  let IconComponent: React.ReactNode;
```

Comment:
You can reduce redunduncy code with below if you like it.

```typescript
  type BottonTabIconMap = {
    [key: string]: {
      [key: string]: React.FunctionComponent<React.SVGAttributes<SVGElement>>;
    };
  };
  const map: BottonTabIconMap = {
    groups: { on: HomeIcon_On, off: HomeIcon_Off },
    foods: { on: FoodIcon_On, off: FoodIcon_Off },
    profile: { on: ProfileIcon_On, off: ProfileIcon_Off },
  };
  // Icon component based on "name" and "isSelected" variable
  const IconComponent: React.ReactNode = (
    <Icon
      icon={map[name][isSelected ? 'on' : 'off']}
      size={size}
      color={color}
      className={className}
    />
  );
```

Reply:
That's kind of similar to what I mentioned, haha
Let's figure out the best solution.
https://github.com/genesis-tech-tribe/nishiki-frontend/pull/87#discussion_r1425916028

Reply:
@uskayyyyy @kotaaaa 
Thank you for all the reviews!

About the unnecessary margins,  I'll ask designers and figure things out.

About the reviews I reacted with 👍 , I'll work on it and fix them immediately.

About the refactor on [this one](https://github.com/genesis-tech-tribe/nishiki-frontend/pull/87#discussion_r1425916028), I think for now, I'll refactor it with Kota's solution.
And later in another scope, as Yusuke mentioned, we can discuss and refactor how we'll organize routes.

@yukiyohure Plz take a look at [this one](https://github.com/genesis-tech-tribe/nishiki-frontend/pull/87#discussion_r1425967577), and let me hear your opinion again 🙏

Reply:
You are mentioning a person who left this project LOL

> And later in another scope, as Yusuke mentioned, we can discuss and refactor how we'll organize routes.
Got it!